falcon/util/uri.py

```diff
@@ -376,3 +376,31 @@ def parse_host(host, default_port=None):
     # or a domain name plus a port
     name, _, port = host.partition(':')
     return (name, int(port))
+
+
+def unquote_string(quoted):
+    """unquote a rfc7320 "quoted-string" into normal one
```

Comment:
Style
* First word in the first line of a comment should be capitalized.
* First line of a comment should end in a period.
* `rfc7320` ==> `RFC 7320`

Grammar
* `a RFC 7320 "quoted-string"` ==> `an RFC 7320 "quoted-string"`
* `into normal one` is superfluous and can be removed from the sentence.